lib/cli.js

```diff
@@ -10,26 +10,85 @@ const resolveFrom = require("resolve-from");
 const standalone = require("./standalone");
 
 /*:: type minimistOptionsType = {
-  default: {
-    config: boolean,
-    f: string,
-    q: boolean,
-  },
-  alias: {
-    f: string,
-    h: string,
-    i: string,
-    id: string,
-    q: string,
-    rd: string,
-    s: string,
-    v: string,
-    aei?: string,
-  },
-  boolean: Array<string>,
+  flags: {
+    "allow-empty-input": {
+      alias: "aei",
+      type: "boolean"
+    },
+    cache: {
+      type: "boolean"
+    },
+    "cache-location": {
+      type: "string"
+    },
+    config: {
+      default: false,
+      type: "string"
+    },
+    "config-basedir": {
+      type: "string"
+    },
+    color: {
+      type: "boolean"
+    },
+    "custom-formatter": {
+      type: "string"
+    },
+    "custom-syntax": {
+      type: "string"
+    },
+    "disable-default-ignores": {
+      alias: "di",
+      type: "boolean"
+    },
+    fix: {
+      type: "boolean"
+    },
+    formatter: {
+      alias: "f",
+      type: "string"
+    },
+    help: {
+      alias: "h",
+      type: "boolean"
+    },
+    "ignore-disables": {
+      alias: "id",
+      type: "boolean"
+    },
+    "ignore-path": {
+      alias: "i"
+    },
+    "ignore-pattern": {
+      alias: "ip"
+    },
+    "no-color": {
+      type: "boolean"
+    },
+    "report-needless-disables": {
+      alias: "rd"
+    },
+    "stdin-filename": {
+      type: "string"
+    },
+    quiet: {
+      alias: "q",
+      type: "boolean",
+      default: false
+    },
+    syntax: {
+      alias: "s"
+    },
+    version: {
+      alias: "v",
+      type: "boolean"
+    }
+  }
```

Comment:
I'm not familiar with Flow, but it feels not right that this object is just a copy of `const minimistOptions`.

Maybe this one is more correct?

```
/*:: type minimistOptionsType = {
  flags: {
    "allow-empty-input": {
      alias: string,
      type: string
    },
    cache: {
      type: string
    },
    "cache-location": {
      type: string
    },
    config: {
      default: boolean,
      type: string
    },
    "config-basedir": {
      type: string
    },
    color: {
      type: string
    },
    "custom-formatter": {
      type: string
    },
    "custom-syntax": {
      type: string
    },
    "disable-default-ignores": {
      alias: string,
      type: string
    },
    fix: {
      type: string
    },
    formatter: {
      alias: string,
      type: string
    },
    help: {
      alias: string,
      type: string
    },
    "ignore-disables": {
      alias: string,
      type: string
    },
    "ignore-path": {
      alias: string
    },
    "ignore-pattern": {
      alias: string
    },
    "no-color": {
      type: string
    },
    "report-needless-disables": {
      alias: string
    },
    "stdin-filename": {
      type: string
    },
    quiet: {
      alias: string,
      type: string,
      default: boolean
    },
    syntax: {
      alias: string
    },
    version: {
      alias: string,
      type: string
    }
  }
}*/
```

Reply:
@hudochenkov you are right, stupid error :smile: